### ISSUE TITLE
???

### ISSUE THREAD

**Issue by DominicanRD13** [lacymorrow/crossover#155]
```
TypeError: Cannot read property 'setBadge' of undefined
    at NsisUpdater.<anonymous> (C:\Users\Carlos Cordero\AppData\Local\Programs\crossover\resources\app.asar\src\main.js:154:14)
    at NsisUpdater.emit (events.js:315:20)
    at NsisUpdater.dispatchUpdateDownloaded (C:\Users\Carlos Cordero\AppData\Local\Programs\crossover\resources\app.asar\node_modules\electron-updater\out\AppUpdater.js:382:14)
    at Object.done (C:\Users\Carlos Cordero\AppData\Local\Programs\crossover\resources\app.asar\node_modules\electron-updater\out\BaseUpdater.js:29:22)
    at done (C:\Users\Carlos Cordero\AppData\Local\Programs\crossover\resources\app.asar\node_modules\electron-updater\out\AppUpdater.js:494:31)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async NsisUpdater.executeDownload (C:\Users\Carlos Cordero\AppData\Local\Programs\crossover\resources\app.asar\node_modules\electron-updater\out\AppUpdater.js:504:20)
```

---

CrossOver 2.7.4
Electron 12.2.3
win32 10.0.19044
Locale: es

**Comment by lacymorrow** [comment]
I'm really sorry about this, I believe I pushed a bug in version 2.7.4 that actually broke the auto-updater. 

The fix is to manually install v3.0.0: https://github.com/lacymorrow/crossover/releases/tag/v3.0.0

**Comment by lacymorrow** [comment]
duplicate #153 #154 #155
